packages/core/src/utils/deserializeNode.tsx

```diff
@@ -71,6 +74,15 @@ export const deserializeNode = (
 ): Omit<NodeData, 'event'> => {
   const { type: Comp, props: Props, ...nodeData } = data;
 
+  invariant(
+    (Comp !== undefined && typeof Comp === 'string' && Comp.length > 0) ||
+      (Comp as { resolvedName?: string }).resolvedName !== undefined,
+    ERROR_DESERIALIZE_COMPONENT_NOT_IN_RESOLVER.replace(
+      '%displayName%',
+      data.displayName
+    ).replace('%availableComponents%', Object.keys(resolver).join(', '))
+  );
```

Comment:
Can we break this line up so it is more readable? 

Ie:
```suggestion
   const isCompHtmlElement = Comp !== undefined && typeof Comp === 'string';
   const isCompUserComponent = Comp !== undefined && (Comp as { resolvedName?: string }).resolvedName !== undefined;

   invariant(isCompHtmlElement || isCompUserComponent,  '....')
```

Reply:
good idea, done